### ISSUE TITLE
[SF][a11y][TextArea] Error state in TextArea valueStateMessage is not announced by screen reader

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents#5716]
### **Bug Description**
Hi Team
Regarding error state announcement of valueStateMessage, thanks for fixing https://github.com/SAP/ui5-webcomponents/issues/5495! But our team also find same issue in `ui5-textarea` valueStateMessage.

### **Expected Behavior**
The valueState should be also announced when `ui5-textarea`  valueStateMessage triggered.

### **Steps to Reproduce**
1. Go to [TextArea document](https://sap.github.io/ui5-webcomponents/playground/components/TextArea/)
2. Trigger out the value state message in any input control
3. Check if the value state is announced by screen reader or not.

### **Context**
- UI5 Web Components version: 1.6.0
- OS/Platform: Any
- Browser: Chrome
- Affected component: `ui5-textarea`

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @ksblklu ,
I could hear the reading of valueStateMessage slotted in ui5-textarea component with JAWS 2022, so in my opinion there is
no issue with it.
Can you, please, provide an example where the issue occurs?

Best regards,
Petya Markova.
(UI5 WC Dispatcher)

**Comment by ksblklu** [comment]
Hi @PetyaMarkovaBogdanova , you can try [this example](https://codesandbox.io/s/text-area-valuestateerror-q87eul) to reproduce
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/27538066/186803508-a4b43372-c028-4ada-b432-0535ad7026b5.png">

**Comment by PetyaMarkovaBogdanova** [comment]
Hi again @ksblklu ,
Thank you for the provided example!
Even with it, I was not able to reproduce the issue. I used Windows 10 Jaws 2022 and NVDA 2022 and the value state message seems to be read out properly. Maybe if the issue persist in your testing, you can record a video and provide more information on screen reader that you used.
Thank you!
Best,
Petya Markova.
(UI5 WC Dispatcher)

**Comment by ksblklu** [comment]
Hi @PetyaMarkovaBogdanova , I made a recording testing this issue, would you please have a look? You can access the code sandbox example in the recording [here](https://codesandbox.io/s/text-area-valuestateerror-q87eul?file=/src/App.js). 

https://user-images.githubusercontent.com/27538066/187380166-6edebe05-b412-4b11-863e-f02aafc40ca1.mp4

**Comment by ilhan007** [comment]
Hello @ndeshev could you dispatch this week, it doesn not have a team assigned yet, thank you!

**Comment by ndeshev** [comment]
Hey @SAP/ui5-webcomponents-topic-rl there is an issue in the `textarea` component - the value state type is not announced - only the value state message text. 

For example - if we have `ui5-textarea` with value state type "Warning" and value state message "This is an important text" - only the "This is an important text" speech output is conveyed. 

This issue was fixed for all other input components with https://github.com/SAP/ui5-webcomponents/pull/5515
It is reproducible everywhere, the easiest way is to test it on our dev test page - https://sap.github.io/ui5-webcomponents/playground/main/pages/TextArea

**Comment by don-obrien** [comment]
Please try to complete by Sept. 15, if possible, as this is higher urgency for SFSF.  Thank you.
